scripts/bump.js

```diff
@@ -0,0 +1,25 @@
+/* global process, require */
+
+const fs = require( 'fs' );
+const shell = require( 'shelljs' );
+
+const args = process.argv;
+
+if ( !( args && args[ 2 ] && args[ 2 ].length > 2 ) ) {
+	console.log( 'Missing CKEditor version! USAGE: npm run bump -- A.B.C, for example: npm run bump -- 4.11.5' );
```

Comment:
It can be changed into guard clause with early exit. Instead of `console.log` we can use `console.error` and then invoke `process.exit( 1 )`.

BTW `--` is totally unnecessary, `npm run bump <version>` works the same.